### PR TITLE
Form page

### PR DESCRIPTION
**Pull request by Quaquaro**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,28 +2,26 @@ import { ErrorBoundary } from 'react-error-boundary';
...
-          ) : (
-            <TrackedGamesList games={games} />
-          )}
-
-          <TrackGameForm onTrackGame={trackGame} />
-        </AppLayout>
+        <Routes>
+          <Route
+            path="/"
+            element={
```
Comment (JelenadlS) [inline src/App.js:19]: Cool idea to give an example!👍🏼

**Review by JelenadlS — APPROVED** [review]
Hey Chris,
I would love to give you more constructive feedback, however I cannot find anything. 
I think it looks really in my opinion! 😊
I will approve it here, but keep it in slack on a black check so that someone else will also check in order to give more feedback - i hope this is fine for you!

**Review by FaGru — APPROVED** [review]
Nice code. I couldn't find any problems.

File: src/components/TrackGameForm.js [hunk 2]
```diff
@@ -94,39 +97,40 @@ export default function TrackGameForm({ onTrackGame }) {
 
 const Form = styled.form`
   padding: 0 8px 0 15px;
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  grid-template-rows: repeat(3, 1fr);
-  grid-column-gap: 8px;
-  grid-row-gap: 8px;
-  place-content: center;
+  @media (max-width: 500px) {
```
Comment (JelenadlS) [inline src/components/TrackGameForm.js:100]: cool that you are using media - i can learn a lot from you😅

**Review by JelenadlS — APPROVED** [review]
Looks all good and more efficient 👍🏼

**Review by ahohnsen — APPROVED** [review]
Looks all good to me :)

**Pull request merged**